CONTRIBUTING.md

```diff
@@ -14,9 +14,7 @@ Please read https://reactjs.org/ and the Code of Conduct before opening an issue
 <a name="bug"/>
 ## Think You Found a Bug?
 
-Please provide a test case of some sort. Best is a pull request with a failing test. Next is a link to codepen/jsbin or repository that illustrates the bug. Finally, some copy/pastable code is acceptable.
-
-If you don't provide a test case, the issue will be closed.
```

Comment:
Maybe? I think it's okay to ask for test cases when they're helpful, but sometimes it's simple enough to not need a test case.